### PR TITLE
[FIX] util.rename_xmlid: also rename `_inherits` xmlids

### PR DESCRIPTION
**Pull request by KangOl**

### CODE REVIEW

File: src/util/records.py [hunk 1]
```diff
@@ -597,6 +597,17 @@ def rename_xmlid(cr, old, new, noupdate=None, on_collision="fail"):
...
+        for inh in inherit_parents(cr, model, recursive=False):
+            if inh.via:
+                parent = inh.model.replace(".", "_")
+                rename_xmlid(
+                    cr,
+                    "{}_{}".format(old, parent),
+                    "{}_{}".format(new, parent),
+                    noupdate=noupdate,
+                    on_collision=on_collision,
+                )
```
Comment (aj-fuentes) [inline src/util/records.py:610]: Do we need the `recursive=False` only for this?
```suggestion
        inh = next(inherit_parents(cr, model), None)
        if inh and inh.via:
            parent = inh.model.replace(".", "_")
            rename_xmlid(
                cr,
                "{}_{}".format(old, parent),
                "{}_{}".format(new, parent),
                noupdate=noupdate,
                on_collision=on_collision,
            )
```

Comment (aj-fuentes) [reply src/util/records.py:610]: Actually from the error in runbot maybe `parent = next(inherit_parents(cr, model), None)` is enough?

Comment (KangOl) [reply src/util/records.py:610]: Nothing forbid models to have multiple `_inherits`.

**Comment by KangOl** [comment]
upgradeci retry with always only crm

File: src/util/records.py (hunk 1 above)
Comment (aj-fuentes) [reply src/util/records.py]: I see...

**Comment by aj-fuentes** [comment]
upgradeci retry

**Review by aj-fuentes — APPROVED** [review]
LGTM

**Comment by aj-fuentes** [comment]
upgradeci retry with always only crm in versions 7.0 8.0 9.0 10.0 11.0

**Comment by KangOl** [comment]
However, it bothers me that I have to change the API of the method.
Maybe another method should be added.

**Review by aj-fuentes — APPROVED** [review]

**Comment by KangOl** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**